tests/syntax-tests/source/Vue/example.vue

```diff
@@ -8,6 +8,11 @@
   </div>
 </template>
 
+<template lang='pug'>
+  #container.col
+    p This shall be formated as Plain Text as long as a Pug syntax definition is missing
```

Comment:
Small typo
```suggestion
    p This shall be formatted as Plain Text as long as a Pug syntax definition is missing
```

Reply:
Thanks, fixed. I wrote that text before I installed a type-checker in VS Code 🙃